_rules/video-strict-alternative-for-visual-1ec09b.md

```diff
@@ -133,29 +110,29 @@ A video element with an incorrect audio description.
 
 #### Failed Example 2
 
-A video element with a track element that contains incorrect descriptions.
+A video element with a link to a text transcript.
```

Comment:
I'm not sure how is that failing the rule. Where is the requirement that a text transcript needs to be on the same page?

Reply:
I think this example is correct. A transcript wouldn't be an acceptable alternative for video with audio.

Reply:
Sorry, got that wrong.